lib/functions/adjust.js

```diff
@@ -26,3 +26,5 @@ module.exports = function adjust(color, prop, amount){
   hsl[prop] += val;
   return hsl.rgba;
 };
+adjust.params = ['color', 'prop', 'amount'];
+module.exports = adjust;
```

Comment:
why that change?

Reply:
When the function source code is minified, function arguments will be renamed and Stylus will fail to extract the correct function arguments:
https://github.com/stylus/stylus/blob/57e260d613d4694b00a0d3d825d7fcfe5210bf2c/lib/utils.js#L375-L379
https://github.com/stylus/stylus/blob/57e260d613d4694b00a0d3d825d7fcfe5210bf2c/lib/visitor/evaluator.js#L1033

By defining a `params` property, we can provide the correct arguments explicitly. [AngularJS also uses this technic to define an argument list](https://docs.angularjs.org/guide/di#-inject-property-annotation).

Reply:
I get that part. I asked on the line where you moved the assignment to `module.exports`.

Reply:
I moved the assignment to a separated line so that the function expression becomes a function definition, then we can reference the function via `adjust`.